### PR TITLE
scaler fitted to the training set only, then applied to test set

### PR DESCRIPTION
**Pull request by nicolasfauchereau**
Modified the boston.py example so that the standard scaler is fitted over the training set, not the whole dataset, and then applied to the test set. The MSE is improved to ~ 18.

### CODE REVIEW

**Comment by nicolasfauchereau** [comment]
I signed the cla !

**Comment by terrytangyuan** [comment]
Thanks!
